docs/getting_started.md

```diff
@@ -3,62 +3,74 @@
 
 ## Prerequisites
 
-1. Python >= 3.6
-2. Configure AWS Account credentials when testing with AWS platforms,
-   check default section in `~/.aws/credentials` for access/secret key
-   [check aws-configuration](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
-3. oc client binary is installed on your localhost and binary is listed in $PATH
-   (running oc version on terminal should display version > 3.11).
-   Latest client can be downloaded from [oc-client](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/).
-4. For vSphere based installations, [terraform](https://learn.hashicorp.com/terraform/getting-started/install.html)
-   and [jq]( https://stedolan.github.io/jq/download/) should be installed ( terraform version should be 0.11.13  )
-5. For UI testing follow the instructions in [openshift console](https://github.com/openshift/console#dependencies) repository.
-   After clone of repository and setting all dependencies please create configuration file with RUN section with `openshift_console_path`
-   parameter pointing to your local clone of openshift console repo. See example in `conf/ocsci/ui-testing.yaml`.
+* Python >= 3.6
+* oc client binary available in the $PATH.
+  (running the oc version on terminal should display version > 3.11).
+  The latest client can be downloaded from [the oc-client public mirror](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/).
+* GNU sed [1]
```

Comment:
[1]  this should be done as anchor link where user can click from here and will be moved directly to that one section when it's at the end of the file.  And you can do it more visible like:  `For mac user see the note [1]`

Reply:
I wanted to use a footnote that some markdown systems support, but apparently github markdown does not (yay, markdown). I can try an improve the structuring some more later once more of the content is improved.